vignettes/dm.Rmd

```diff
@@ -14,9 +14,7 @@ editor_options:
 source("setup/setup.R")
 ``````
 
-## Introduction
```

Comment:
Removing this section header to be consistent with most of the other vignettes that don't have it.